docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -47,6 +57,22 @@ If there is an error please [contact support](mailto:pulsesupport@codacy.com).
 -   The set of changes in a deployment is the list of commits in the corresponding pull request.
 -   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
 
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The following is a detailed description of how the Pulse Bitbucket integration automatically detects incidents:
+
+### Use pull request reverts (based on default branch)
+
+-   Pulse considers an incident a reverted pull request that **targets the default branch** of the repository when the revert operation was executed through the Bitbucket UI originating a branch which name starts with `revert-`.
```

Comment:
```suggestion
-   Pulse considers an incident a pull request that **targets the default branch** of the repository when the revert operation was executed through the Bitbucket UI originating a branch which name starts with `revert-pr-`.
```

I find that "reverted" is a bit strange there.

Also, **any** PR being merged from a branch with prefix `revert-pr-` to the default branch will be considered an incident, not just reverts that happen through the UI.

I think it would also be nice to point users to [this Bitbucket documentation page](https://support.atlassian.com/bitbucket-cloud/docs/merge-a-pull-request/#Revert-a-merged-pull-request) where they can learn how to revert a PR.

Reply:
> I find that "reverted" is a bit strange there.
do. you think it's not clear enough? or that it doesn't reflect what we use?

Reply:
I updated this section considering all the input. 👍